fnlb/lb/allgrouper.go

```diff
@@ -48,6 +48,12 @@ func NewAllGrouper(conf Config, db DBStore) (Grouper, error) {
 	return a, nil
 }
 
+const (
+	UNKNOWN   = iota
+	HEALTHY   = iota
+	UNHEALTHY = iota
```

Comment:
go nit, would make a type so it's obvious what this is (vs an int) and also consts in go are generally camel cased (as unfortunate as it may be, it is the idiom)

```go
type state uint8

const (
  StateUnknown state = iota
  StateHealthy
  StateUnhealthy
)
```